frontend/keysets/simple-urdu-keyset/simple-urdu-keys-layout.ts

```diff
@@ -0,0 +1,31 @@
+export const unshiftedKeysLayout = [
+  ["`", "۱", "۲", "۳", "۴", "۵", "٦", "۷", "۸", "۹", "۰", "-", "=", "Delete"],
+  ["Tab", "ق", "و", "ع", "ر", "ت", "ے", "ء", "ی", "ہ", "پ", "[", "]", "\\"],
+  ["Caps", "ا", "س", "د", "ف", "گ", "ح", "ج", "ک", "ل", "؛", "'", "Enter"],
+  ["Shift", "ز", "ش", "چ", "ط", "ب", "ن", "م", "،", "۔", "/", "Shift"],
+  ["Ctrl", "Alt", "Space", "Alt", "Ctrl"],
+];
+
+export const shiftedKeysLayout = [
+  ["~", "!", "@", "#", "٪", "ﷲ", "ٔ", "،", "*", ")", "(", "_", "+", "Delete"],
+  ["Tab", "ٰ", "ُ", "ٗ", "ڑ", "ٹ", "ۓ", "ّ", "َ", "ۂ", "ﷺ", "{", "}", "|"],
+  ["Caps", "آ", "ص", "ڈ", "", "غ", "ح", "ژ", "خ", "ُ", ":", "\"", "Enter"],
+  ["Shift", "ذ", "ً", "ٔ", "ظ", "ب", "ں", "؟", ">", "ّ", "؟", "Shift"],
```

Comment:
```suggestion
  ["~", "!", "@", "#", "$", "٪", "^", "&", "*", "(", ")", "ٰ", "_", "+", "Delete"],
  ["Tab", "ؒ", "ؤ", "ؓ", "ڑ", "ٹ", "ۓ", "ؑ", "ئ", "ۂ", "ؐ", "ٗ", "{", "}", "|"],
  ["Caps", "آ", "ص", "ڈ", "أ", "غ", "ھ", "ض", "خ", "ۃ", "ً", ":", '"', "Enter"],
  ["Shift", "ذ", "ژ", "ث", "ظ", "<", "ں", ">", "ٍ", ".", "٫", "؟", "Shift"],
```
- remove duplicates
- change the positions according to phonetic layout
- add new keys for diacritics (right of alphabet, left of symbols)
- add missing characters
- remove unnormalized characters
- fix parentheses order (change the div's direction to RTL instead of swapping the order here)